eth/tracers/internal/tracetest/makeTest.js

```diff
@@ -44,10 +47,15 @@ var makeTest = function(tx, traceConfig) {
         context.baseFeePerGas = block.baseFeePerGas.toString();
     }
 
-    console.log(JSON.stringify({
+    var data = {
         genesis: genesis,
         context: context,
-        input:  eth.getRawTransaction(tx),
-        result: result,
-    }, null, 2));
+        input:   eth.getRawTransaction(tx),
+        result:  result,
+    };
+    if (traceConfig) {
+        data.tracerConfig = traceConfig;
```

Comment:
Hm I think this is wrong. traceConfig is here something like `{ tracer: 'callTracer', traceConfig: {...}}`. We should get the nested traceConfig.

Reply:
Oh yeah, you're right, I forgot this one.